docs/intro/version_history.rst

```diff
@@ -1,6 +1,74 @@
 Version history
 ---------------
 
+1.3.0
+=====
+
+* As of this release, we now have an official :repo:`breaking change policy
+  </CONTRIBUTING.md#breaking-change-policy>`.
+* Bazel is now the canonical build system (replacing CMake). There have been a huge number of
+  changes to the development/build/test flow. See :repo:`/bazel/README.md` and
+  :repo:`/ci/README.md` for more information.
+* :ref:`Outlier detection <arch_overview_outlier_detection>` has been expanded to include success
+  rate variance, and all parameters are now configurable in both runtime and in the JSON
+  configuration.
+* TCP level :ref:`listener <config_listeners_per_connection_buffer_limit_bytes>` and
+  :ref:`cluster <config_cluster_manager_cluster_per_connection_buffer_limit_bytes>` connections now
+  have configurable buffer limits at which point connection level back pressure is applied. Full end
```

Comment:
We technically don't have back pressure yet, we're only using them on the receive side to ensure we start pushing the buffers down the filter chain when we hit the limits.